modules/local/filter_stats.nf

```diff
@@ -0,0 +1,28 @@
+process FILTER_STATS {
+    label 'process_medium'
+
+    conda (params.enable_conda ? 'bowtie2=2.4.5' : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/bowtie2:2.4.5--py39hd2f7db1_2' :
+        'quay.io/biocontainers/bowtie2:2.4.5--py36hfca12d5_2' }"
```

Comment:
Why does this use the bowtie2 image?

Reply:
I think the main reason is as the bowtie2 image is already used in smrnaseq, that we skip downloading a different  / base image for only this process in general and instead use any other image.